app/forms/nominate_induction_tutor_form.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class NominateInductionTutorForm
+  include ActiveModel::Model
+
+  attr_accessor :full_name, :email, :token
+
+  validates :full_name, presence: { message: "Enter a name" }, on: :details
+  validates :email, presence: { message: "Enter email" }, on: :details
+
+  def save!
+    school = NominationEmail.find_by(token: token).school
+
+    user = ActiveRecord::Base.transaction do
+      user_record = User.create!(
+        full_name: full_name,
+        email: email,
+        confirmed_at: Time.zone.now,
+      )
+
+      InductionCoordinatorProfile.create!(user: user_record, schools: [school])
+      user_record
+    end
+
+    InviteSchools.new.run [school.urn]
```

Comment:
This is wrong, right? This should be the `nomination_confirmation_email`

Reply:
You're right it should be the nomination_confirmation_email